content/blog/2023/03/10/2023-03-09-jenkins-newsletter.adoc

```diff
@@ -0,0 +1,156 @@
+Before it is fully deprecated, we'll also release a merged version of the `centos` and `centos7` image as the very last CentOS `7` Docker image.
+
+In regards to our Docker images, there were several updates here as well:
+
+* The latest updates are now part of the agent images such as:
+** ssh-agent: Upgraded Git version on Windows to 2.39.2.windows.1 (https://github.com/jenkinsci/docker-ssh-agent/pull/209[#209]) @github-actions
+** docker-agent: Upgraded Git version on Windows to 2.39.2.windows.1 (https://github.com/jenkinsci/docker-agent/pull/376[#376]) @github-actions
+** Inbound agent:
+*** Upgraded the parent image jenkins/agent version to 3107.v665000b_51092-4 (https://github.com/jenkinsci/docker-inbound-agent/pull/331[#331]) @github-actions
+*** Upgraded the parent image jenkins/agent version to 3107.v665000b_51092-3 (https://github.com/jenkinsci/docker-inbound-agent/pull/330[#330]) @github-actions
+*** Upgraded updatecli/updatecli-action from 2.19.0 to 2.20.1 (https://github.com/jenkinsci/docker-inbound-agent/pull/329[#329]) https://github.com/dependabot[@dependabot]
+*** The Windows controller image is not updated as often as the rest.
+It's been more than one year without any updates, and we may choose to drop it.
+
+* With the release of Debian 12 (“bookworm”), Debian will no longer deliver OpenJDK 11.
+** Thankfully, the end of life date for Debian's openJDK11 won't happen until 2026 or 2027.
+** The Jenkins documentation will be updated when it goes out, so that we describe the use and installation of Jenkins with openJDK17.
+
+New platforms:
+
+* RISC-V support is far from official for Jenkins, but tests are progressing.
+
+[[modern-ui]]
+image:/images/post-images/2023/01/12/jenkins-newsletter/ui_ux.png[User Experience Update]
+
+Contributed by: author:markewaite[Mark Waite]
+
+User experience improvements continued to arrive in February, thanks to contributions from author:janfaracik[Jan Faracik], author:notmyfault[Alexander Brandes], author:timja[Tim Jacomb], https://github.com/mawinter69[Markus Winter], and others.
+Look for the improvements in recent weekly releases and in the new Jenkins `2.387.1` LTS release.
+
+The plugin:pipeline-graph-view[pipeline graph viewer plugin] continues to improve its user interface.
+Refer to the video highlights in the https://community.jenkins.io/t/user-experience-sig-march-1-2023/6057[[.underline]#User Experience SIG recording#].
+Additionally, build logs are now viewed from the main panel with easier navigation.
+
+[[security-fixes]]
+image:/images/post-images/2023/01/12/jenkins-newsletter/security.png[Security Update]
+
+Contributed by: author:wadeck[Wadeck Follonier]
+
+The security team has prepared the March 8, 2023 security release of Jenkins `2.394` (weekly), Jenkins `2.387.1` (LTS), and Jenkins `2.375.4` (previous LTS).
+Special thanks to Daniel Beck and other security team members for their work on the upcoming release of Jenkins core and our plugin security reports.
```

Comment:
```suggestion
Contributed by: author:kguerroudj[Kevin Guerroudj]

Two security advisories have been published during the month of February:

* One about link:/security/advisory/2023-02-15/[plugins], 5 plugins were affected and have been fixed, with one vulnerable to a sandbox bypass vulnerability.

* One about link:/security/advisory/2023-02-09/[Docker images], fixing the vulnerabilities https://github.com/git/git/security/advisories/GHSA-c738-c5qq-xg89[CVE-2022-23521] and https://github.com/git/git/security/advisories/GHSA-475x-2q3q-hvwq[CVE-2022-41903] present in git allowing to achieve remote code execution.

The security team recommends users to update as soon as possible.
```

I think we should focus on what happened during February there, also I guess we shouldn't pretend to be Wadeck, I updated my profile there if you can merge it https://github.com/jenkins-infra/jenkins.io/pull/6131